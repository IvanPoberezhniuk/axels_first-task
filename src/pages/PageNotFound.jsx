import React from 'react';

const PageNotFound = props => {
  return (
    <div>
      <h2>404</h2>
    </div>
  );
};

PageNotFound.propTypes = {};

export default PageNotFound;
