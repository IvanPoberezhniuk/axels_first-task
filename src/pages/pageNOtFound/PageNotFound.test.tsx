import { shallow, ShallowWrapper } from 'enzyme';

import PageNotFound from './PageNotFound';

let component: ShallowWrapper;
const setUp = () => shallow(PageNotFound());

beforeEach(() => {
  component = setUp();
});

describe('PageNotFound component', () => {
  it('should take snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});

