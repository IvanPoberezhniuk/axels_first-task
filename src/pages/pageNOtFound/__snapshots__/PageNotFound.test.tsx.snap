// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component SNAPSHOT 1`] = `
<div>
  <h2>
    404
  </h2>
</div>
`;
